__tests__/app.test.js

```diff
@@ -49,3 +49,42 @@ describe("/api", () => {
       });
   });
 });
+
+describe("/api/articles/:article_id", () => {
+  it("GET responds with status code 200", () => {
+    return request(app).get("/api/articles/2").expect(200);
+  });
+  it("GET responds with status code 200", () => {
+    return request(app)
+      .get("/api/articles/2")
+      .then((response) => {
+        expect(response.body.article[0]).toMatchObject({
+          author: expect.any(String),
+          title: expect.any(String),
+          article_id: expect.any(Number),
```

Comment:
You need to check the article_id is the one you've requested i.e. 2. This would match for any article at the moment and you couldnt be sure you've got back the correct one.